packages/wonder-blocks-dropdown/components/single-select-menu.md

```diff
@@ -0,0 +1,114 @@
+### Single select menu. Regular and light versions.
+
+```js
+const {View} = require("@khanacademy/wonder-blocks-core");
+const {StyleSheet} = require("aphrodite");
+
+const styles = StyleSheet.create({
+    row: {
+        flexDirection: "row",
+    },
+    sideMargins: {
+        marginRight: 10,
+    },
+    darkBackgroundWrapper: {
+        background: "black",
+        padding: 10,
+    },
+    strutLike: {
+        width: 8,
+    }
+});
+<View style={[styles.row]}>
+    <SingleSelectMenu
+        items={[
+            {
+                type: "select",
+                label: "Banana wrapped in a peel ",
+                selected: false,
+                value: "banana"
+            },
+            {
+                type: "select",
+                label: "Apple",
+                selected: false,
+                value: "apple"
+            },
+            {
+                type: "select",
+                label: "Grape",
+                selected: false,
+                value: "grape"
+            },
+        ]}
+        light={false}
+        onChange={(s) => console.log(s)}
```

Comment:
Users of this component will probably want to use the current selection later so they'll need to store it in `state` somewhere.  Since they're already storing it in state it makes sense to pass it as a prop to `SingleSelectMenu` so that it doesn't get out of sync with what `SingleSelectMenu` is displaying.  Can you update this example to define a parent component that tracks the selected value just like devs would have to do to use this this for real?

Reply:
We may also want to replace the `select` property on each of the `items` with a single `value` prop on `SingleSelectMenu` itself.  There are a couple of components that do this, e.g. https://jedwatson.github.io/react-select/, https://www.npmjs.com/package/react-select.